fixtures/blockers.py

```diff
@@ -49,3 +50,47 @@ def bug(blocker):
         Instance of :py:class:`utils.bz.BugWrapper` or :py:class:`NoneType` if the bug is closed.
     """
     return lambda bug_id: blocker("BZ#{}".format(bug_id)).bugzilla_bug
+
+
+def pytest_addoption(parser):
+    group = parser.getgroup('Blockers options')
+    group.addoption('--disable-blocker-listing',
```

Comment:
I like the ability to list blockers, but I think it's good to have this disabled by default. How about `--list-blockers`?